holonote/annotate/display.py

```diff
@@ -489,7 +491,7 @@ def static_indicators(self, **events):
             "data": self.data,
             "region_labels": region_labels,
             "fields_labels": fields_labels,
-            "invert_axes": False,  # Not yet handled
+            "invert_axes": self.invert_axis,
```

Comment:
I suggest adding a small comment here that this is only been looked at for range1d.